src/pages/PhoneSignIn.vue

```diff
@@ -89,6 +95,32 @@ export default {
       if (num_match != null) return true;
       return false;
     },
+
+    async handleClickGoogleSignIn(){
+      try {
+        const googleUser = await this.$gAuth.signIn();
+        if (!googleUser) {
+          return null;
+        }
+        this.user = googleUser.getBasicProfile().getEmail();
+        let socialAuthToken = googleUser.getAuthResponse();
+        await this.convertSocialAuthToken(socialAuthToken.access_token);
+        this.$router.push({name: 'Home'});
+      } catch (error) {
+        console.error(error);
+        return null;
+      }
+    },
+  },
+  setup(props) {
+    const { isSignIn } = toRefs(props);
+    const Vue3GoogleOauth = inject("Vue3GoogleOauth");
+    const handleClickLogin = () => {};
+    return {
+      Vue3GoogleOauth,
+      handleClickLogin,
+      isSignIn,
+    };

```

Comment:
can we please avoid using the Composition API and instead use the Options API used everywhere else in the code?

Reply:
@dalmia as per my understanding (after research), Composition API was introduced to address limitations in Options API. Do you think we should use Options API?
![image](https://user-images.githubusercontent.com/12053186/113889879-466e4680-97e1-11eb-80d6-94d30accbe32.png)

Full article here:
https://auth0.com/blog/getting-started-with-vue-3-composition-api/

Reply:
The only reason for not using Composition API is that our codebase currently uses only Options API through - having this particular piece in Composition API format is just a bit out of place given that - it would have totally made sense if our codebase was written in Composition API itself.

Reply:
@dalmia while trying to convert it into Options API, I figured we don't need this code right now. It's mostly for button's reactivity to keep it enabled/disabled and we're anyway using router.beforeEach to make sure people don't login again.

Since this file was updated in the phone OTP Pull Request, I've removed the code in that branch.